sway/ipc-server.c

```diff
@@ -35,58 +33,53 @@
 #include "log.h"
 #include "util.h"
 
-static int ipc_socket = -1;
-static struct wl_event_source *ipc_event_source =  NULL;
-static struct sockaddr_un *ipc_sockaddr = NULL;
+struct ipc {
+	int socket;
+	struct wl_event_source *event_source;
+	struct sockaddr_un *sockaddr;
+};
+
+static struct ipc sway_ipc = { .socket = -1 };
+static struct ipc i3_ipc = { .socket = -1 };
+
 static list_t *ipc_client_list = NULL;
 static struct wl_listener ipc_display_destroy;
 
 static const char ipc_magic[] = {'i', '3', '-', 'i', 'p', 'c'};
```

Comment:
Would it make sense to keep using this magic number for I3SOCK, but use something like "sway-ipc" for SWAYSOCK?

Reply:
nah